src/Compilers/CSharp/Test/Semantic/Semantics/UnsafeTests.cs

```diff
@@ -944,6 +950,44 @@ public void UnsafeIteratorSignatures()
                 Diagnostic(ErrorCode.ERR_IllegalInnerUnsafe, "Iterator")); //this is for putting "unsafe" on an iterator, not for the parameter type
         }
 
+        [Fact]
+        public void UnsafeIteratorSignatures_PointerArray()
```

Comment:
>UnsafeIteratorSignatures_PointerArray [](http://example.com/codeflow?start=20&length=37)

The other comment was placed by CodeFlow in a wrong place. In the test ```UnsafeIteratorSignatures_PointerArray``` (commit 1 vs. 9), we are removing:
```
                // (4,66): error CS1637: Iterators cannot have unsafe parameters or yield types
                //      System.Collections.Generic.IEnumerable<int> Iterator(int*[] p)
                Diagnostic(ErrorCode.ERR_UnsafeIteratorArgType, "p").WithLocation(4, 66)
```

Perhaps the `unsafe` is a misleading term to use in the message now. The check is specifically about pointer types now, and `unsafe` types in a more general term are not disallowed.